android/sdk/src/main/java/com/taobao/weex/http/Options.java

```diff
@@ -216,13 +216,23 @@
   private Map<String, String> headers;
   private String body;
   private Type type = Type.text;
+  private int timeout = DEFAULT_TIMEOUT;
```

Comment:
change to `timeoutMillis`,  I mean add 'Millis' suffix to these field/arguments.